docs/units/equivalencies.rst

```diff
@@ -418,6 +418,16 @@ between the Celsius, Fahrenheit, and Kelvin. For example::
 
 .. note:: You can also use ``u.deg_C`` instead of ``u.Celsius``.
 
+Mass-energy Equivalency
+-----------------------
+
+In a special relativity context, mass and energy can be equivalent units. For
+example::
+
+    >>> import astropy.units as u
```

Comment:
```suggestion
    >>> from astropy import units as u
```

Reply:
Why the change here?

Reply:
I thought `from ... import ... as ...` is the standard practice, no?

Reply:
Okay, I see that this is just following other examples. Ignore this then.